src/lib/Microsoft.Health.Events/EventConsumers/Service/EventConsumerService.cs

```diff
@@ -35,44 +48,52 @@ public async Task ConsumeEvents(IEnumerable<IEventMessage> events)
             {
                 foreach (IEventConsumer eventConsumer in _eventConsumers)
                 {
-                    await OperationWithRetryAsync(eventConsumer, events);
+                    try
+                    {
+                        await RetryPolicy.ExecuteAsync(async () => await eventConsumer.ConsumeAsync(events).ConfigureAwait(false));
```

Comment:
I wonder if it makes more sense to move this code to the actual event consumer - In this case it would be the Microsoft.Health.Fhir.Ingest.Console.Normalize.Processor class, specifically in the ConsumeAsync() method. It appears that, considering the proposed changes in Startup.cs, we only want these changes in the normalization processor. Moving the retry logic there would encapsulate it to just normalization, and would mean we would not need to update the workspace-platform code.

Reply:
Great point; I'll go ahead and do that.

Reply:
Done